v21.1/cockroach-debug-list-files.md

```diff
@@ -0,0 +1,81 @@
+---
+title: cockroach debug list-files
+summary: Learn the command for listing the files collected in the debug zip.
+toc: true
+key: debug-list-files.html
+---
+
+The `cockroach debug list-files` [command](cockroach-commands.html) shows the files that will be collected by using [`cockroach debug zip`](cockroach-debug-zip.html).
+
+{{site.data.alerts.callout_info}}
+The files listed include logs, heap profiles, and goroutines. Other [files](cockroach-debug-zip.html#files) generated by `cockroach debug zip` are not listed by `cockroach debug list-files`.
```

Comment:
"goroutine dumps". Goroutines, on their own, are a run-time construct and do not correspond to files.

Reply:
Also, CPU profiles (see my comment below).